projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.scss

```diff
@@ -20,53 +16,52 @@
   .document-tree-invisible {
     display: none;
   }
-  
+
   .document-tree ul,
   .document-tree li {
     margin-top: 0;
     margin-bottom: 0;
     list-style-type: none;
   }
-  
+
   .document-tree-node {
     display: block;
   }
-  
+
   .document-tree-node .document-tree-node {
     padding-left: 40px;
   }
-  
-  .node {
-    display: flex;
-    padding: 4px;
-  }
-  
-  .icon {
-    width: 36px;
-    height: 36px;
-    background-size: 100%;
-    border: 0px;
-    background-color: #FFF;
-  }
-  
-  .icon-folder {
-    background-image: url(/assets/images/folder.png);
-  }
-  
-  .icon-document {
-    background-image: url(/assets/images/document.png);
+}
+
+.node {
+  display: flex;
+  align-items: center;
+  padding: 6px 0;
+
+  &__icon {
+    position: relative;
+    background: none;
+    border: 0;
+    padding: 6px;
   }
-  
-  .node-name {
-    margin-left: 6px;
+
+  &__iconImg {
+    display: block;
+    height: 24px;
+    width: 24px;
   }
-  
-  .node-name-folder,
```

Comment:
Some styles are missing. Please let me know whether the UI looks good and as expected without these missing styles.

Reply:
It looks good.